example/Podfile

```diff
@@ -6,7 +6,7 @@ target 'ios_example' do
   use_frameworks!
 
   # Pods for ios_example
-  pod 'WechatOpenSDK', '~> 2.0.4'
+  pod 'WechatOpenSDK-XCFramework', '~> 2.0.4'
```

Comment:
Please include this link as comment above this line. https://developers.weixin.qq.com/community/develop/doc/000c62b9b0ccf8c1d65fa90b751001?blockType=1

From the link, it says that we need to change the header include macro from `""` to `<>`. I wonder why we did not do so.

For the removal of the `.a` and `.h` files, I guess Cocoapods has handled that for us. So no action is required from us.

Reply:
> From the link, it says that we need to change the header include macro from "" to <>. I wonder why we did not do so.

It is because I found that `""` seems also working. Let me change it back to `<>` according to the doc.